### PR TITLE
increase plasmacutter price. reduce anglegrinder price

### PR DESCRIPTION
**Pull request by Erikafox**
:cl:
balance: plasma cutters r more expensive.
balance: angle grinders are cheaper, and come with a bowman headset.
/:cl:

### CODE REVIEW

**Comment by tantspants** [comment]
angle grinders suck

File: code/modules/cargo/packs/tools.dm [hunk 1]
```diff
@@ -112,8 +105,11 @@
 /datum/supply_pack/tools/anglegrinder
 	name = "Angle Grinder"
 	desc = "Contains one angle grinder pack, a tool used for quick structure deconstruction and salvaging"
-	cost = 2000
-	contains = list(/obj/item/gear_pack/anglegrinder)
+	cost = 1500
+	contains = list(
+		/obj/item/gear_pack/anglegrinder
```
Comment (Erikafox) [inline code/modules/cargo/packs/tools.dm]:
```suggestion
		/obj/item/gear_pack/anglegrinder,
```

**Comment by Erikafox** [comment]
i have become the webeditor

**Comment by generalthrax** [comment]
What's the pitch behind this one?

**Comment by Erikafox** [comment]
> What's the pitch behind this one?

well. anglegrinders need ear protection (not being in the crate is an oversight from me i think) and plasma cutters are a better version of angle grinders. i would be open to just making them cost like. 2500~ 3000~ credits but being available.

**Comment by generalthrax** [comment]
I was thinking of making a pr myself actually that would've made anglegrinders more efficient and cheaper than plasmacutters but plasmacutters are portable and more expensive.

Grinders are the shipbreaker's choice and cutters are a favourite for thieves, pirates, military crews.

**Comment by Erikafox** [comment]
I think these ideas can co-exist.

Plasma cutters being a either specialist, or black market tool, and grinders being an effective but bulky solution.

Imo its in the design direction for them that I keep in the back of my head.

**Comment by generalthrax** [comment]
I think the move here is to make plasmacutters more expensive (the price you suggested, they're just industrial tools it probably shouldnt go in the black market) and do the rest of what you did with the angle grinders. I'll probably do the Other Shit between now and next uptime.

2500's probably good

**Comment by Erikafox** [comment]
yeah sure let me update the pr

**Pull request merged**